src/components/browser-banner/_macro.spec.js

```diff
@@ -5,83 +5,84 @@ import * as cheerio from 'cheerio';
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
 
-const EXAMPLE_BROWSER_BANNER_MINIMAL = {};
+const EXAMPLE_BROWSER_BANNER_PARAMS = {
+  bannerLeadingText: 'This website no longer supports your browser.',
+  bannerCTA:
+    'You can <a class="ons-browser-banner__link" href="https://www.ons.gov.uk/help/browsers">upgrade your browser to the latest version</a>.',
+  bannerLinkUrl: 'https://www.ons.gov.uk/help/browsers',
+};
 
 describe('macro: browser-banner', () => {
-  it('passes jest-axe checks with', async () => {
-    const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_MINIMAL));
+  describe.each([
+    ['no parameters provided', {}],
+    ['all parameters provided', EXAMPLE_BROWSER_BANNER_PARAMS],
```

Comment:
These tests would still pass even if the implementation was removed since the defaults use the same values.